src/foam/nanos/crunch/predicate/StatusChangedFrom.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
Why can't we do this with fscript? 
n.status!=o.status 
and can be combined with checking some particular status
o.status==...&&n.status!=o.status  
for example.

Reply:
I need it for a rule predicate. Can we use fcript in this case?

Reply:
yes, see foam.nanos.crunch.invoke-edit-behaviour for an example: 
        class: 'foam.nanos.ruler.predicate.FScriptRulePredicate',
        query: 'o.status==foam.nanos.crunch.CapabilityJunctionStatus.GRANTED'